mapie/classification.py

```diff
@@ -0,0 +1,437 @@
+from __future__ import annotations
+from typing import Optional, Union, Iterable, Any, cast
+
+import numpy as np
+from sklearn.utils import check_X_y, check_array
+from sklearn.utils.validation import check_is_fitted
+from sklearn.base import clone
+from sklearn.base import BaseEstimator, ClassifierMixin
+from sklearn.linear_model import LogisticRegression
+from sklearn.model_selection import BaseCrossValidator, KFold, LeaveOneOut
+from sklearn.pipeline import Pipeline
+
+from ._typing import ArrayLike
+from .utils import check_null_weight, fit_estimator
+
+
+class MapieClassifier (BaseEstimator, ClassifierMixin):  # type: ignore
+    """
+    TO BE CONTINUED
```

Comment:
Je rajouterais une première ligne de description du type : 
"Prediction intervals for classification using conformal predictions (in development)."